### PR TITLE
Update check_process.default

### PR DESCRIPTION
**Pull request by kay0u**

### CODE REVIEW

File: check_process.default [hunk 1]
```diff
@@ -27,18 +27,8 @@
 		port_already_use=0
```
Comment (kay0u) [inline check_process.default]: Perhaps we should remove `incorrect_path=1`?

Comment (maniackcrudelis) [reply check_process.default]: Yep, I remember that we were supposed to.
But, we should find where's the commit that check that.
And add it here as a justification.

Comment (kay0u) [reply check_process.default]: Got it!

https://github.com/YunoHost/yunohost/pull/304

**Comment by kay0u** [comment]
https://github.com/YunoHost/package_check/pull/56

**Comment by kay0u** [comment]
Bump

**Pull request merged**